File: extract.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('TDR Forum Juin.pdf');

pdf(dataBuffer).then(function(data) {
    fs.writeFileSync('tdr_text.txt', data.text);
    console.log('TDR Done.');
});

let dataBuffer2 = fs.readFileSync('gt Forum année passée.pdf');

pdf(dataBuffer2).then(function(data) {
    fs.writeFileSync('gt_text.txt', data.text);
    console.log('GT Done.');
});

File: extract.py
from pypdf import PdfReader

reader = PdfReader("TDR Forum Juin.pdf")
text = ""
for page in reader.pages:
    text += page.extract_text() + "\n"

with open("tdr_text.txt", "w", encoding="utf-8") as f:
    f.write(text)

reader2 = PdfReader("gt Forum année passée.pdf")
text2 = ""
for page in reader2.pages:
    text2 += page.extract_text() + "\n"

with open("gt_text.txt", "w", encoding="utf-8") as f:
    f.write(text2)

print("Extraction complete.")
